apps/web/components/booking/BookingListItem.tsx

```diff
@@ -507,9 +523,11 @@ function BookingListItem(booking: BookingItemProps) {
               )}
               {booking.attendees.length !== 0 && (
                 <DisplayAttendees
-                  attendees={booking.attendees}
+                  attendees={attendeeList}
                   user={booking.user}
                   currentEmail={userEmail}
+                  bookingUid={booking.uid}
+                  isPast={isPast}
```

Comment:
nit: Can we rename this prop to isBookingInPast or something i had to dig around to see what isPast meant in this context.